iceoryx_posh/include/iceoryx_posh/iceoryx_posh_types.hpp

```diff
@@ -72,6 +72,7 @@ constexpr char SHM_NAME[] = "/iceoryx_mgmt";
 using namespace units::duration_literals;
 
 // Timeout
+constexpr units::Duration PROCESS_FINAL_KILL_TIME = 45_s;
```

Comment:
Name could be revised... it's not immediately clear what this means without context.
Maybe: `PROCESS_MAXIMUM_KILL_DELAY` or something. 

Maybe a bit pedantic.

Reply:
That's right. I renamed it and all related variables, incl. the command line option.